articles/index.yml

```diff
@@ -99,7 +99,10 @@ landingContent:
         links:
           - text: Standard libraries
             url: libraries/standard/index.md
-          - text: Numerics library
-            url: libraries/numerics/index.md
           - text: Chemistry library
             url: libraries/chemistry/index.md
+          - text: Machine learning library
```

Comment:
```suggestion
          - text: Quantum machine learning library
```
For consistency with the TOC

Reply:
To be consistent with how the TOC and the cards are different. The shorter version as is more concise and may be better?

![image](https://user-images.githubusercontent.com/59890648/76987453-797a3980-6943-11ea-8196-a118a91198cf.png)

Reply:
Ok, you're right! Sorry